crates/sargon/Cargo.toml

```diff
@@ -24,11 +24,17 @@ log = "0.4.20"
 # pretty_env_logger = "0.5.0"
 pretty_env_logger = { git = "https://github.com/seanmonstar/pretty-env-logger/", rev = "0e238400e18649415dc710c025e99c009a1bb744" }
 
-# derive_more = "1.0.0-beta.6"
-derive_more = { git = "https://github.com/JelteF/derive_more", rev = "1196b2dd7a366c06db621093884adbc379fc0f0a", features = [
+# derive_more = "1.0.0"
+derive_more = { git = "https://github.com/JelteF/derive_more", rev = "d7f5b9e94d024790682f6fc4dcca13941cce64c8", features = [
+  "add",
```

Comment:
Derive more has become very powerful! We should use these ! `as_ref` and `deref` are very nice!